src/index.js

```diff
@@ -49,6 +49,10 @@ export function createI18n (options = { messages: {}, locale: 'en', wikilinks: t
       app.provide(contextSymbol, bananai18n)
 
       app.config.globalProperties.$i18n = (msg, params) => {
+        params = params || []
+        if (!Array.isArray(params)) {
+          params = [params]
+        }
         return bananai18n.i18n(msg, params)
```

Comment:
Here also, you need `...params`. Please add the following test to see it fails
```javascript

describe('Vue-Banana-i18n global $i18n with mutliple params', () => {
  let wrapper
  beforeEach(() => {
    wrapper = mount({
      template: `
          <p>{{$i18n(msg, params)}}</p>
        `,
      props: {
        msg: { type: String, default: 'empty-results' },
        params: { type: Array }
      }
    }, {
      props: {
        msg: 'empty-results',
        params: ['Invalid page', 'English']
      },
      global: {
        plugins: [i18n]
      }
    })
  })
  it('will correctly replace all placeholders while using $i18n', () => {
    assert.strict.equal(wrapper.text(), 'No pages found for "Invalid page" in English')
  })
})
```

Reply:
Fixed in following commit.